tasks/section3.yml

```diff
@@ -1033,8 +1033,8 @@
 - name: "NOTSCORED | 3.7 | Disable IPv6"
   replace:
       dest: /etc/default/grub
-      regexp: '(^GRUB_CMDLINE_LINUX\s*\=\s*)(?:")(.+)(?<!ipv6.disable=1)(?:")'
-      replace: '\1"\2 ipv6.disable=1"'
+      regexp: '^(GRUB_CMDLINE_LINUX=(?!.*ipv6.disable)\"[^\"]*)(\".*)'
+      replace: '\1 ipv6.disable=1\2'
```

Comment:
It looks like the regex is not perfect. If ipv6.disable=0 is there it will not be replaced.